scripts/select_msoas.py

```diff
@@ -1,12 +1,37 @@
 # This script helps you define new study areas. You can write the output to a
-# new file, config/some_region.csv .
-
+# new file, config/some_region.csv.
```

Comment:
The second sentence is incorrect now, the script writes output for you

Reply:
Ok.